src/Content/Conversation.php

```diff
@@ -408,6 +408,11 @@ public function statusEditor(array $x = [], int $notes_cid = 0, bool $popup = fa
 
 			'$compose_link_title'  => $this->l10n->t('Open Compose page'),
 			'$always_open_compose' => $this->pConfig->get($this->session->getLocalUserId(), 'frio', 'always_open_compose', false),
+			// Dropzone
+			//'$max_imagesize'       => DI::config()->get('system', 'maximagesize'),
+			// DI::config.. does not work here, so it is set to a manual value
```

Comment:
In this file you should use `$this->config->get('system', 'maximagesize')`, but `DI::config()->get('system', 'maximagesize')` should also work.